packages/garbo/src/fights.ts

```diff
@@ -398,6 +399,14 @@ function familiarSpec(underwater: boolean, fight: string): OutfitSpec {
   }
 
   if (isFreeAndCopyable(globalOptions.target)) {
+    if (["Macrometeorite", "Powerful Glove"].includes(fight)) {
```

Comment:
```suggestion
    if (fight.gregariousReplace) {
```

Reply:
This doesn't actually work and requires passing more than we are today to define the fight.